File: 백준/Bronze/1330. 두 수 비교하기/두 수 비교하기.js
const input = require('fs').readFileSync('/dev/stdin').toString().trim().split(' ');

const A = parseInt(input[0]);
const B = parseInt(input[1]);

if (A > B) {
  console.log('>');
} else if (A < B) {
  console.log('<');
} else {
  console.log('==');
}

File: 백준/Bronze/1330. 두 수 비교하기/두 수 비교하기.kt
fun main(){
    val input = readln().split(" ").map{ it.toInt() }
    
    val A = input[0]
    val B = input[1]    
    
    if (A > B) {
        println(">")
    } else if (A < B) {
        println("<")
    } else if (A === B) {
        println("==")
    }
}